e2e/src/main/java/org/jetbrains/bsp/bazel/BazelBspSampleRepoTest.java

```diff
@@ -255,14 +255,20 @@ private BazelBspTestScenarioStep dependencySourcesResults() {
             "https/repo1.maven.org/maven2/com/google/guava/guava/28.0-jre/guava-28.0-jre-sources.jar");
 
     DependencySourcesItem exampleExampleDependencies =
-        new DependencySourcesItem(new BuildTargetIdentifier("//example:example"), dependencies);
+        new DependencySourcesItem(
+            new BuildTargetIdentifier("//example:example"), ImmutableList.of());
```

Comment:
Maybe we can get rid of guava altogerher. I am heavily using vavr in the core of sync. I replaced ImmutableList.of in this file, but I'd rather do such refactorings on a separate PR to avoid conflicts.

Reply:
cool, vavr for the win